__tests__/app.test.js

```diff
@@ -0,0 +1,37 @@
+const app = require("../app.js");
+const request = require("supertest");
+const db = require("../db/connection.js");
+const seed = require("../db/seeds/seed.js");
+const {
+  articleData,
+  commentData,
+  topicData,
+  userData,
+} = require("../db/data/test-data/index.js");
+
+beforeEach(() => seed({ topicData, userData, articleData, commentData }));
+afterAll(() => db.end());
+
+describe("GET /api/topics", () => {
+  it("Returns an array", () => {
```

Comment:
This test could be superfluous as your next test will fail if an array is not returned